src/components/ec-dropdown-search/ec-dropdown-search.story.js

```diff
@@ -102,4 +110,52 @@ stories.add('in container with a dynamic width', () => ({
   `,
 }));
 
+stories.add('with custom item template', () => ({
+  components: { EcDropdownSearch, EcIcon },
+  data() {
+    return {
+      selectedItem: null,
+    };
+  },
+  props: {
+    items: {
+      default: object('Items', items),
+    },
+    isSearchEnabled: {
+      default: boolean('isSearchEnabled', true),
+    },
+    paragraphText: {
```

Comment:
Can be merged: This is needed? I mean, there are no changes from the basic, just the custom slot right?

Reply:
The text is there only to demonstrate that dropdown floats over the content and not pushing it down.